### PR TITLE
Created postgres data model

### PR DESCRIPTION
**Pull request by dpaikoff**
@NickTAdams can you review my pr?

- Created bulk insert function
  - Created fake groups row function
  - Created fake events row function
- Created select to json sql statement
  - Created function to template select to json sql statement
- Created test suite to test db functions

### CODE REVIEW

**Comment by NickTAdams** [comment]
@dpaikoff :  
I think the code is clean, if dense. It is hard for me to understand what's happening under the hood with the  pgp library, though I don't think it's a fault of the code. I would spend some time nailing down what each of the functions in this library are doing so that you can explain it to another (senior) engineer so that they can understand what is happening with the data flow here.

Good separation of concerns, no function seems to bleed into what others should be doing.

Data flow happens in a logical functional programming style.

Good use of testing, I like how generation of data and 10k data rows are tested.

tldr; Ready to merge.

-Nick
